test/util/benchmark_results.cpp

```diff
@@ -367,13 +367,13 @@ void BenchmarkResults::save_results()
     std::string baseline_file = m_results_file_stem;
     std::string latest_csv_file = m_results_file_stem + ".latest.csv";
     baseline_file += ".baseline";
-    int r;
     if (!util::File::exists(baseline_file)) {
-        r = link(name.c_str(), baseline_file.c_str());
+        int r = link(name.c_str(), baseline_file.c_str());
+        static_cast<void>(r); // FIXME: Display if error
     }
     if (util::File::exists(latest_csv_file)) {
-        r = unlink(latest_csv_file.c_str());
+        (void)unlink(latest_csv_file.c_str());
```

Comment:
Not a huge deal, but why not add a FIXME the same way as `link` above? Then if we ever come back to handle the errors here, we can do them all the same way (points for consistency).